uber/models.py

```diff
@@ -1182,14 +1182,14 @@ def last_first(cls):
     @property
     def watchlist_guess(self):
         try:
-            with sa.Session() as session:
-                return session.guess_attendee_watchentry(self)
+            with Session() as session:
+                return [u.__dict__ for u in session.guess_attendee_watchentry(self)]
```

Comment:
Hm, Stack overflow misrepresented what `__dict__ `is, then. 

IIRC, using it here ends up returning an empty dictionary.